services/core-web/src/components/common/DocumentTable.js

```diff
@@ -84,96 +139,178 @@ const withTag = (text, elem) => {
 
 export const DocumentTable = (props) => {
   const isMinimalView = props.view === "minimal";
-
-  let columns = [
-    {
-      title: "Name",
-      key: "name",
-      dataIndex: "name",
-      sorter: !isMinimalView && nullableStringSorter("name"),
-      render: (text, record) => {
-        let content = record.name;
-
-        if (!isMinimalView) {
-          content = (
-            <div key={record.key} title="Name">
-              <DocumentLink
-                documentManagerGuid={record.document_manager_guid}
-                documentName={record.name}
-                truncateDocumentName={false}
-              />
+  let columns = props.expandable
+    ? [
+      {
+        className: "file-name-column",
+        title: "File Name",
+        dataIndex: "fileName",
+        key: "fileName",
+        render: (text, record) => {
+          return (
+            <div className="file-name-container" title="File Name">
+              <span className="file-name-text" style={!record.numberOfVersions ? { marginLeft: "23px" } : {}}>{text}</span>
+              {record?.numberOfVersions > 0 ?
+                (<span className="file-history-container">
+                  <span>
+                    <Tooltip
+                      title={`This file has ${record.numberOfVersions} previous versions`}
+                      placement="top"
+                      mouseEnterDelay={1}
+                    >
+                      <span className="file-version-amount">
```

Comment:
Would this one be a candidate for the [Tag](https://ant.design/components/tag) antd component? Used it for the Archived tag in the same table, and think it would be good to keep it consistent if possible 🙃

Reply:
Hi Simen, this has been adjusted now, I changed `withTag` function a bit.